src/sentry/static/sentry/app/components/events/interfaces/utils.jsx

```diff
@@ -66,8 +66,13 @@ export function stringifyQueryList(query) {
   }
 
   const queryObj = {};
-  for (const [k, v] of query) {
-    queryObj[k] = v;
+  for (const kv of query) {
+    if(kv !== null && kv.length == 2) {
+      const [k, v] = kv;
+      if(v !== null) {
+        queryObj[k] = v;
+      }
+    }
```

Comment:
Should we add a test for this situation? Or does updating the data file reflect in existing tests?

Reply:
Yeah the data file is for Percy tests.